trinity/cli_parser.py

```diff
@@ -189,31 +184,3 @@ def __call__(self,
         "Enables profiling via cProfile."
     ),
 )
-
-
-#
-# Add `console` sub-command to trinity CLI.
-#
-console_parser = subparser.add_parser(
-    'console', help='run the chain and start the trinity REPL')
```

Comment:
Maybe add a comment in this config file that plugins may add subcommands that set a `func` attribute to be run instead of the main node?